DirectProgramming/C++SYCL_FPGA/Tutorials/Features/multi_aocx/device_link/src/vector_ops.h

```diff
@@ -0,0 +1,23 @@
+#ifndef VECTOR_OPS_H
+#define VECTOR_OPS_H
+
+#include <sycl/sycl.hpp>
+#include <array>
+
+using namespace sycl;
+// ARRAY type & data size for use in this example
+constexpr size_t array_size = 10000;
```

Comment:
Do we need such a large size? It will slow down simulation - can we make it smaller?

Reply:
Do you think 100 would be reasonable?